adr/ADR-8.md

```diff
@@ -318,9 +382,9 @@ While the cache is still warming up it's a pass through to the underlying backen
 Most operations are pass-through, except:
 
  * `Get()` will read local cache if ready (fully warmed), and a key is there
- * `Delete()` will evict the key from local cache and pass through
+ * `Delete()` and `Purge()` will evict the key from local cache and pass through
  * `Put()` will evict the key from local cache and pass through
- * `Destroy()` and `Purge()` will delete all messages from local cache and pass through
+ * `Destroy()` will delete all messages from local cache and pass through
```

Comment:
I'm wondering if we should make local caches an option to the context. Local cache will require a watch running, which means we must have a background thread / async subscription. Maybe someone won't want or need this if they are know they are close to the server.

Reply:
it's opt in